vectorstore_test.go

```diff
@@ -18,6 +18,11 @@ func TestLocalVectorStore_Query(t *testing.T) {
 		t.Fatalf("err: %v\n", err)
 	}
 
+	// Serialize back to disk.
+	if err := store.StoreJSON("testdata/saved_olympics_section.json"); err != nil {
+		t.Fatalf("err: %v\n", err)
+	}
```

Comment:
Strictly speaking, neither `LoadJSON` nor `StoreJSON` has its own unit test currently.

I guess a better way is to add two unit tests for them. But I can add the tests later if you don't have bandwidth :)

Reply:
@RussellLuo - ok, I was piggy-backing off of this existing unit-test but I'll remove this and let you unit-test the Load/Save how you see fit. Thanks!